### ISSUE TITLE
Founder age inaccuracy

### ISSUE THREAD

**Issue by ConflictFan** [daredloco/SincUltimate#17]
**Describe the bug**
A clear and concise description of what the bug is:
Founder starts at 50 years old and not 40

**To Reproduce**
Steps to reproduce the behavio**u**r:
1. Launch game as normal (blablabla)
2. Open employee window
3. Notice founder is 50 and not 40
4. Become depressed

**Expected behaviour**
Founder should be 40, not 50.

**Screenshots**
![image](https://github.com/daredloco/SincUltimate/assets/119769869/6bd62d11-6d18-4205-83de-445913eb2693)

**Software Inc Version:**
Beta 1.7.32

**Mod Version:**
1.2.0

**Additional context**

**Comment by daredloco** [comment]
Strange, in my tests they always were 40 years old. 
```cs
if (founder.employee.GetAge() < 40)
{
    DevConsole.Console.Log($"Changing age of founder \"{founder.employee.Name}\" to 40 years");
    var diffAge = 40 - founder.employee.GetAgeFlat();
    founder.employee.BirthDate = new SDateTime(founder.employee.BirthDate.Month, founder.employee.BirthDate.Year - diffAge);
}
```

To be fair, I did write the code when I was sleepy. I will take a look at it and fix it for v1.2.1

**Comment by daredloco** [comment]
After looking at the code I ask myself: "Why the hell did I do it this way?"
Update is pushed to v1.2.1 branch in a second.
